lib/biran/erb_config.rb

```diff
@@ -1,18 +1,19 @@
 module Biran
   class ERBConfig
-    attr_reader :output_dir, :source_dir, :name, :extension, :config
+    attr_reader :output_dir, :source_dir, :name, :extension, :config, :output_name
```

Comment:
From what I can see these are used to access the variables from in the instance methods. So like in the `save!` method, there is direct access to the `output_dir` and now `output_name` variables/methods.